src/github.com/stellar/gateway/bridge/handlers/request_handler_builder.go

```diff
@@ -40,9 +41,15 @@ func (rh *RequestHandler) Builder(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	sequenceNumber, err := strconv.ParseUint(request.SequenceNumber, 10, 64)
-	if err != nil {
-		errorResponse := protocols.NewInvalidParameterError("sequence_number", request.SequenceNumber, "Sequence number must be a number")
+	if request.SequenceNumber == "" {
```

Comment:
@bartekn  This can also be expressed like below, but this has more lines of code.
What do you think?
```if request.SequenceNumber == "" {
		accountResponse, err := rh.Horizon.LoadAccount(request.Source)
		if err != nil {
			errorResponse := protocols.NewInvalidParameterError("sequence_number", accountResponse.SequenceNumber, "Could not load sequenceNumber from network")
			log.WithFields(errorResponse.LogData).Error(errorResponse.Error())
			server.Write(w, errorResponse)
			return
		}

		sequenceNumber, err = strconv.ParseUint(accountResponse.SequenceNumber, 10, 64)
		if err != nil {
			errorResponse := protocols.NewInvalidParameterError("sequence_number", accountResponse.SequenceNumber, "Could not parse sequence number from network")
			log.WithFields(errorResponse.LogData).Error(errorResponse.Error())
			server.Write(w, errorResponse)
			return
		}
	}else{

		sequenceNumber, err = strconv.ParseUint(request.SequenceNumber, 10, 64)
		if err != nil {
			errorResponse := protocols.NewInvalidParameterError("sequence_number", request.SequenceNumber, "Sequence number must be a number")
			log.WithFields(errorResponse.LogData).Error(errorResponse.Error())
			server.Write(w, errorResponse)
			return
		}
	}```